src/sdk.js

```diff
@@ -250,7 +252,21 @@ class Smartcar {
     link += `&mode=${this.mode}`;
 
     if (options.singleSelect !== undefined) {
-      link += `&single_select=${options.singleSelect === true}`;
+      let singleSelectParamAdded = false;
```

Comment:
Validate keys sent to the function and also add single_select=true if they pass in vin